clinica/iotools/converters/genfi_to_bids/genfi_to_bids_utils.py

```diff
@@ -75,6 +82,20 @@ def filter_dicoms(df: DataFrame) -> DataFrame:
     return df
 
 
+def _handle_series_description(x: str) -> str:
+    series_description = pdcm.dcmread(x).SeriesDescription
```

Comment:
By curiosity, what is the type of `series_description` ?

Reply:
`SeriesDescription` is a string